### PR TITLE
[candi] Ubuntu 24.04 support

### PR DESCRIPTION
**Pull request by borg-z**
## Description
Added support for Ubuntu 22.04

## Why do we need it, and what problem does it solve?
This update ensures compatibility with the latest version of Ubuntu LTS

## Why do we need it in the patch release (if we do)?
Not necessarily.

## What is the expected result?
The system should now correctly detect and install necessary packages for Ubuntu 22.04

## Checklist
- [ ] The code is covered by unit tests.
- [x] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [ ] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: be | candi
type: feature
summary: |
  Add support for Ubuntu 22.04. Support for Ubuntu 22.04 ensures compatibility with the latest OS version, providing updated packages and configurations.
impact_level: default
```

### CODE REVIEW

File: candi/bashible/common-steps/node-group/000_discover_node_ip.sh.tpl [hunk 1]
```diff
@@ -67,10 +67,17 @@ function is_ip_in_cidr() {
   test $((netmask & ip_dec)) -eq $((netmask & net_address_dec))
 }
 
-if bb-is-ubuntu-version? 22.04 || bb-is-ubuntu-version? 20.04 || bb-is-ubuntu-version? 18.04; then
```
Comment (borg-z) [inline candi/bashible/common-steps/node-group/000_discover_node_ip.sh.tpl:70]: looks clunky, do we really need this check?

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @borg-z. I've started the workflow for label 'e2e/run/gcp'...

**Review by sprait — APPROVED** [review]
we also need to remember to update `node_os_image_metrics()` in the flant-intergation module hook  (which is currently being moved to an external module).

**Review by z9r5 — APPROVED** [review]

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/test'...

**Review by RomanenkoDenys — APPROVED** [review]

**Pull request merged**